.github/workflows/changelog_checker.yml

```
@@ -0,0 +1,36 @@
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'resources/CHANGELOG.md'
+
+jobs:
+  check-changelog:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)